awx/main/management/commands/register_queue.py

```diff
@@ -95,11 +80,6 @@ def register(self):
                 elif not created:
                     print("Instance Group already registered {}".format(ig.name))
 
-                if self.controller:
-                    (ig_ctrl, changed2) = self.update_instance_group_controller(ig)
```

Comment:
You should get rid of the other uses of `changed2` here. It should work as it is, but we can get rid of the now-useless variable.

Reply:
This is all very much a work in progress.  Reviews are premature.